{{cookiecutter.project_slug}}/requirements/test.txt

```diff
@@ -2,5 +2,5 @@
 tox==2.6.0
 coverage==4.1
 {% if cookiecutter.use_pytest == 'y' -%}
-pytest==3.0.7
+pytest >=3.8
 pytest-cov==2.4.0{%- endif %}
```

Comment:
I guess at this point it might be worth updating pytest-cov as weel. Do you have time to test it please? :penguin: